### DISCUSSION TITLE
Concernant l'execution time

### DISCUSSION THREAD

**Discussion by CamilleTCH** [heig-vd-dai-course/heig-vd-dai-course#433] [Q&A] [answered]
Nous nous demandions si l'exécution time doit être écrit dans la console ou dans notre fichier ?

![image](https://github.com/user-attachments/assets/342d6902-01d1-4a3a-9b49-d7099e3e88e3)

**Comment by ludelafo** [comment, marked as answer]
L'affichage dans la console est amplement suffisant, comme dans le template du chapitre Java IOs!
